### ISSUE TITLE
Freezing, dont do anything (2.2.0)

### ISSUE THREAD

**Issue by PFFyodor** [fireice-uk/xmr-stak#706]
latest version
works better, make amd.txt and cpu.txt, before i dont see this files,
works in some iterations, when i try different parameters to tune perfomance, when i left one car ( i have 2) in config, i get freesing:
freezing - dont do anything, dont closing or killing and pc not rebooting, needs hot shutdown PC

i change only amd.txt

[amd.txt](https://github.com/fireice-uk/xmr-stak/files/1586144/amd.txt)
[config.txt](https://github.com/fireice-uk/xmr-stak/files/1586145/config.txt)
[cpu.txt](https://github.com/fireice-uk/xmr-stak/files/1586146/cpu.txt)

one more problem - perfomance, its strange, rx460 card gets goo speed, like Claymore 10.2, but RX470 get only about 620 H/s, (710 H/s in Claymore 10.2), changing untesity and worksize parameters dont get positive results, only negative.

Currently its in freesing, and i cant make hard reboot PC now, for continue testing, becourse need work.

**Comment by JerichoJones** [comment]
This is a bit hard to follow so I'm going to see if you agree with my interpretation and we can go from there.

- You are running v2.2.0
- amd.txt and cpu.txt are now created successfully.
- You have two GPUs (RX460 & RX470?)
- If you remove one of them from amd.txt and run STAK the PC freezes (not just STAK)
- You then have to hard reset the PC.

If we agree to the above I have some questions:
1. What operating system and version?
2. What GPU driver and version?
3. Which card do you remove from the amd.txt? (RX460 or RX470)
4. What happens when you reverse what you did in 3. ? Does the PC freeze?
5. Delete amd.txt let STAK recreate it. DO NOT MAKE ANY CHANGES.
6. Perform steps 3 & 4 above. Report back the results.
